tests/scripts/tests-setup.sh

```diff
@@ -3,6 +3,19 @@
 
 set -x
 
+PODMAN_BACKEND_CONFIG_FILE="/etc/containers/containers.conf.d/50-network-backend.conf"
+if [ "$GITHUB_ACTIONS" == "true" ]; then
+    expected_backend="netavark"
+    podman_network_backend=$(sudo podman info --format "{{.Host.NetworkBackend}}")
+
+    if [ "${podman_network_backend}" != "${expected_backend}" ]; then
+        echo "Setting Host Network Backend as netavark..."
+        echo "[network]" > "${PODMAN_BACKEND_CONFIG_FILE}"
```

Comment:
I think you need to `mkdir /etc/containers/containers.conf.d` first because it might not exist.

Reply:
Thanks @Luap99